src/backend/gl/src/conv.rs

```diff
@@ -132,9 +132,7 @@ pub fn describe_format(format: Format) -> Option<FormatDescription> {
             4,
             Float,
         ),
-        Bgra8Unorm => {
-            FormatDescription::new(glow::RGBA8, glow::BGRA, glow::UNSIGNED_BYTE, 4, Float)
-        }
+        Bgra8Unorm => FormatDescription::new(glow::BGRA, glow::BGRA, glow::UNSIGNED_BYTE, 4, Float),
```

Comment:
looks like `Bgra8Srgb` is still a problem

Reply:
Yes, but I'm not sure which internal format is correct for Bgra8Srgb.